README.md

```diff
@@ -266,11 +287,22 @@ file.name.prefix=some-prefix/
 # Optional, the default is `none`.
 file.compression.type=gzip
 
+# Specifies the current timezone in which the dates and times will be treated.
+# Accepts long and short standard names like: `UTC`, `PST` and `ECT` or
+# `Europe/Berlin`, `Europe/Helsinki` or `America/New_York`. 
+# For more information please read documentation: https://docs.oracle.com/javase/tutorial/datetime/iso/timezones.html.
+# The defauilt is `UTC`
+file.name.timestamp.timezone=Europe/Berlin
+
+# Determines how to obtain a timestamp.
+# Supports only `wallclock` which is default value
+file.name.timestamp.source=wallclock
+
 # The file name template.
 # See "File name format" section.
-# Optional, the default is `{{topic}}-{{partition}}-{{start_offset:padding=false}}` or
+# Optional, the default is `{{topic}}-{{partition}}-{{start_offset:padding=false}}-{{timestamp}}` or
```

Comment:
This doc change is incorrect according to the code (and we shouldn't change this default without a strong reason as well).

Reply:
true